src/modules/deal-investor/dto/deal-investor-response.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { ApiProperty, ApiResponseProperty } from '@nestjs/swagger';
+import { DateString } from '@ukef/helpers/date-string.type';
+
+class DealInvestorDtoLenderType {
+  @ApiProperty({ description: 'Lender type: 100 for Exporter or 500 for UKEF record.', example: '500' })
+  LenderTypeCode: string;
+}
+export class DealInvestorResponseDto {
+  @ApiResponseProperty({ example: '0030000321' })
+  dealIdentifier: string;
+
+  @ApiResponseProperty({ example: 'E1' })
+  portfolioIdentifier: string;
```

Comment:
I think we do need `Enum` here to ensure only specific set of values can be passed.

Reply:
Added enum and constant